docs/PowerAuth-SDK-for-Android.md

```diff
@@ -867,75 +856,44 @@ if (powerAuthSDK.hasValidActivation()) {
-                    powerAuthSDK.removeActivationLocal(context);
-                    break;
-                case ActivationStatus.State_Deadlock:
-                    // Local activation is technically blocked and no longer
-                    // can be used for the signature calculations. You can inform
-                    // user about this situation and remove activation locally.
-                    android.util.Log.i(TAG, "Activation is technically blocked");
-                    powerAuthSDK.removeActivationLocal(context);
-                    break;
-                case ActivationStatus.State_Created:
-                    // Activation is just created. This is the internal
-                    // state on the server and therefore can be ignored
-                    // on the mobile application.
-                default:
-                    android.util.Log.i(TAG, "Unknown state");
-                    break;
-            }
 
-            // Failed login attempts, remaining = max - current
-            int currentFailCount = status.failCount;
-            int maxAllowedFailCount = status.maxFailCount;
-            int remainingFailCount = status.getRemainingAttempts();
+Note that the status fetch may fail at an unrecoverable error `PowerAuthErrorCodes.PROTOCOL_UPGRADE`, meaning that it's not possible to upgrade the PowerAuth protocol to a newer version. In this case, it's recommended to [remove the activation locally](#activation-removal).
 
-            if (status.getCustomObject() != null) {
-                // Custom object contains any proprietary server specific data
-            }
-        }
+### Activation states
 
-        @Override
-        public void onActivationStatusFailed(Throwable t) {
-            // Network error occurred, report it to the user
-        }
-    });
-} else {
-    // No activation present on device
-}
-```
-<!-- end -->
+This chapter explains activation states in detail. To get more information about activation states, check the [Activation States](https://github.com/wultra/powerauth-crypto/blob/develop/docs/Activation.md#activation-states) chapter available in our [powerauth-crypto](https://github.com/wultra/powerauth-crypto) repository.
 
-Note that the status fetch may fail at an unrecoverable error `PowerAuthErrorCodes.PROTOCOL_UPGRADE`, meaning that it's not possible to upgrade the PowerAuth protocol to a newer version. In this case, it's recommended to [remove the activation locally](#activation-removal).
+#### `ActivationStatus.State_Created` 
+
+The activation record is created using an external channel, such as the Internet banking, but the key exchange between the client and server did not happen yet. This state is never reported to the mobile client.
+
+#### `ActivationStatus.State_Pending_Commit`
+
+The activation record is created and key exchange between the client and server already took place, but the activation record needs additional approval before it can be used.
```

Comment:
I would add explicitly that it needs to be approved on the server (for example confirmed in the Internet banking or a back office system).